pandas/core/tools/datetimes.py

```diff
@@ -782,15 +782,7 @@ def to_datetime(
         `strftime documentation
         <https://docs.python.org/3/library/datetime.html
         #strftime-and-strptime-behavior>`_ for more information on choices, though
-        note the following differences:
-
-        - :const:`"%f"` will parse all the way
-          up to nanoseconds;
-
-        - :const:`"%S"` without :const:`"%f"` will capture all the way
-          up to nanoseconds if present as decimal places, and will also handle
-          the case where the number of seconds is an integer.
-
+        note that "%f" will parse all the way up to nanoseconds.
```

Comment:
No need to remove the formatting
```suggestion
        note that :const:`"%f"` will parse all the way up to nanoseconds.
```

Reply:
Thank you, I have re-added the formatting.